Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/CrowdStrikeFalcon.py

```diff
@@ -3240,14 +3247,14 @@ def fetch_events():
     last_run = demisto.getLastRun()
     demisto.debug(f"CrowdStrikeFalconMsg: Current last run object is {last_run}")
     if not last_run:
-        last_run = [{}, {}, {}, {}, {}, {}, {}]
-    last_run = migrate_last_run(last_run)
+        last_run = [{}, {}, {}, {}, {}, {}]
+    last_run = migrate_last_run(last_run, is_fetch_events=True)
     current_fetch_info_detections: dict = last_run[0]
     current_fetch_info_incidents: dict = last_run[1]
     current_fetch_info_idp_detections: dict = {} if len(last_run) < 3 else last_run[2]
-    current_fetch_info_mobile_detections: dict = {} if len(last_run) < 6 else last_run[5]
-    current_fetch_on_demand_detections: dict = {} if len(last_run) < 7 else last_run[6]
-    current_fetch_ofp_detection: dict = {} if len(last_run) < 8 else last_run[7]
+    current_fetch_info_mobile_detections: dict = {} if len(last_run) < 4 else last_run[3]
```

Comment:
Risky changing the indexes, Why do we need to change?
Also can we use constants with event types instead of just numbers?